airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -24,6 +24,42 @@
 Changelog
 ---------
 
+7.2.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Add option to wait for completion on the EmrCreateJobFlowOperator (#28827)``
+* ``Add transfer operator S3 to (generic) SQL (#29085)``
+* ``add retries to stop_pipeline on conflict (#29077)``
+* ``Add log for AWS Glue Job Console URL (#28925)``
+* ``Enable individual trigger logging (#27758)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``fix: 'num_of_dpus' typehints- GlueJobHook/Operator (#29176)``
+* ``Fix false-positive spellcheck failure (#29190)``
```

Comment:
#29190 I guess it fix only our internal CI process. I guess I could be classified as Misc or even could exclude in changelog.